engineering_tools/opendrive_lanelet2_converter/opendrive2lanelet2convertor.py

```diff
@@ -0,0 +1,169 @@
+import os
+import sys, getopt
+from pyproj import Proj, transform
+
+from commonroad.common.file_writer import CommonRoadFileWriter
+from opendrive2lanelet.opendriveparser.parser import parse_opendrive
+from opendrive2lanelet.network import Network
+
+import xml.etree.ElementTree as xml
+import xml.dom.minidom as pxml
+from lxml import etree
+
+# class representing node in lanelet2
+class Node:
+    def __init__(self, id, lat, lon, local_x, local_y):
+        self.id = id
+        self.lat = lat
+        self.lon = lon
+        self.local_x = local_x
+        self.local_y = local_y
+
+    def create_xml_node_object(self):
+        node_element = xml.Element('node',{'id': str(self.id), 'lat': str(self.lat), 'lon': str(self.lon), 'version': str(1), 'visible': 'true'})
+        xml.SubElement(node_element, "tag", {"k": "ele", "v": "0.0"})
+        xml.SubElement(node_element, "tag", {"k": "local_x", "v": str(self.local_x)})
+        xml.SubElement(node_element, "tag", {"k": "local_y", "v": str(self.local_y)})
+        return node_element
+
+# class representing way in lanelet2
+class Way:
```

Comment:
A 'way' is not a concept in lanelet2. I believe this refers to a 'way' in OSM which corresponds to a lanelet edge I think. Update the comment to make that clearer.

Reply:
if you look at the lanelet2 file you can see an element called way. that then it used to create relation element with a type of lanelet2.

Reply:
Yes but that element is an OSM concept and not a lanelet2 concept. So it is present in the map file but will not exist once the map is loaded (at least not by that name).

Reply:
yeah this is convertor not a map loader, I think you are confused by the comment `# class representing way in lanelet2` ,  let me change them.